suprb/suprb.py

```diff
@@ -154,6 +154,13 @@ def fit(self, X: np.ndarray, y: np.ndarray, cleanup=False):
             if self.logger_ is not None:
                 self.logger_.log_iteration(X, y, self, iteration=self.step_)
 
+            # Abort if no rules could be found and set fitness to 0
+            if not self.pool_:
+                self.elitist_ = self.solution_composition_.elitist().clone()
+                self.elitist_.fitness_ = 0.0
+                self.is_fitted_ = True
+                return self
+
```

Comment:
Do we still need this? If I understood it correctly there will always be at least one rule within the pool

Reply:
Agreed, forgot to remove this previously